android/engine/src/main/java/org/smartregister/fhircore/engine/util/SharedPreferenceKey.kt

```diff
@@ -32,4 +32,5 @@ enum class SharedPreferenceKey {
   USER_INFO,
   CARE_TEAM,
   ORGANIZATION,
+  DATA_MIGRATION
```

Comment:
Use `PreferenceDataStore` instead. We are migrating to using DataStore instead of shared Preferences.

Reply:
fixed